app/controllers/ems_common.rb

```diff
@@ -834,6 +837,8 @@ def get_form_vars
     @edit[:new][:bearer_token] = params[:bearer_token] if params[:bearer_token]
     @edit[:new][:bearer_verify] = params[:bearer_verify] if params[:bearer_verify]
 
+    @edit[:new][:service_account] = params[:service_account] if params[:service_account]
+
```

Comment:
Ah thank you! 

Yes, the migrations were giving me trouble - how do set that up?